media/css/mozorg/contribute/contribute.scss

```diff
@@ -2,176 +2,396 @@
+        display: flex;
+        flex-direction: column;
+
+        &:hover,
+        &:focus {
+            transform: translateZ(0) scale(1.02);
+            backface-visibility: hidden;
+            transition: transform 0.1s ease-in-out,
+                -webkit-transform 0.1s ease-in-out;
         }
     }
 
-    @media #{$mq-md} {
-        @include clearfix;
+    .contribute-card-content {
+        flex: 1;
+        padding: $spacing-md;
+    }
 
-        .contribute-stats-item {
-            width: 33.3%;
-            @include bidi((
-                (float, left, right),
-            ));
-        }
+    .contribute-card-content-tags {
+        margin-bottom: $spacing-sm;
+        min-height: $layout-md;
     }
-}
 
-//* -------------------------------------------------------------------------- */
-// Main Hero
+    .contribute-card-tag-bold {
+        font-weight: bold;
+    }
+}
 
-.contribute-hero {
-    margin-bottom: $layout-md;
+.contribute-banner-wrapper {
+    max-width: 1420px;
+    min-width: 304px;
+    margin: 0 auto;
+    box-sizing: border-box;
 
     @media #{$mq-md} {
-        margin-bottom: $layout-lg;
+        padding: 0;
+    }
+
+    @media #{$mq-lg} {
+        padding: $layout-xs $layout-xl;
+        padding-bottom: 0;
+        margin-bottom: $layout-xl;
     }
 }
 
-//* -------------------------------------------------------------------------- */
-// Contribution Video
+.contribute-banner {
+    padding-top: $layout-xl;
+    padding-bottom: $layout-md;
+    padding-left: $layout-md;
+    padding-right: $layout-md;
 
-.is-modern-browser .contribute-video {
-    display: none;
-}
+    @media #{$mq-lg} {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        min-height: 260px;
+        position: relative;
+        padding-top: 0;
+        padding-bottom: 0;
+    }
 
-.mzp-c-modal video {
-    display: block;
-    margin: 0 auto;
-    width: 100%;
+    .contribute-banner-content {
+        @media #{$mq-lg} {
+            margin-left: 200px;
+            width: 60%;
+            max-width: 495px;
+        }
+    }
+
+    .contribute-banner-cta {
+        flex: 1 0 auto;
+
+        @media #{$mq-lg} {
+            margin-left: $layout-xl;
+        }
+    }
 }
 
-//* -------------------------------------------------------------------------- */
-// Mozillian Stories Links
+.contribute-banner-gethelp {
+    background-image: url("/media/img/contribute/contribute-gethelp-banner-bg.jpg");
+    background-size: cover;
+    color: $color-white;
 
-.contribute-mozillian-list {
-    margin-top: $layout-lg;
+    @media #{$mq-lg} {
+        margin-bottom: $layout-lg;
+    }
 
-    .contribute-mozillian-item {
-        @include border-box;
-        @include text-body-lg;
-        margin-bottom: $spacing-lg;
-        padding: 0 $spacing-md;
+    .contribute-banner-gethelp-title {
+        font-family: Inter, X-LocaleSpecific, sans-serif;
+        @include font-size(24px);
```

Comment:
Unfortunately it looks like the `@include text-display-md` mixin renders the font at 40px and 2.5rem instead of the 24px that is outlined in the design.

Reply:
It's 24px in small viewports and bumps up to 40px at medium sizes, which I thought is what you had here already, unless I'm misreading.
https://github.com/mozilla/protocol/blob/master/src/assets/sass/protocol/includes/mixins/_typography.scss#L79

Or if the design calls for 24px on larger screens that would be `text-display-xs` (which will be 18px on small screens).
https://github.com/mozilla/protocol/blob/master/src/assets/sass/protocol/includes/mixins/_typography.scss#L97

Reply:
Unfortunately it calls for 24px on all screen sizes, I took a peek and didn't see any existing mixins for that, so explicitly put 24px across all viewports.

Reply:
Ah, I see. In that case, yes, we can't use a mixin and this is fine.

Reply:
Though in the media query below you're increasing to 40px at `mq-lg` (1024px) and our mixins do that at `mq-md` (768px) so I'm not sure I see much difference, apart from just shifting the breakpoint. This may be a case where I'd argue for correcting the design. But this, too, is not a hill worth dying on right now so let's leave it alone.